### DISCUSSION TITLE
Unable to Add a Zoom Meeting ID

### DISCUSSION THREAD

**Discussion by cong862023** [sircharlo/jw-launcher#239] [Q&A]
Hello! 
First of all, excellent app.
I'm facing a particular issue that prevents me from inputting the meeting IDs for my congregation. In this case, my ID is 9XX XXX XXXX. However, when I enter it into the corresponding field, it disappears because apparently it's expecting a "longer" ID.
I'm sending a demonstrative video of this issue. 
In the video I'm using the ID 111 111 1111, which has the same number of characters as mine.

https://github.com/sircharlo/jw-launcher/assets/138789928/bd01e491-1bd9-462b-b508-e5781960287b

**Comment by ScrubMcDuff** [comment]
add 000000000, 9 zeros infront of your meeting ID. still works here.

**Comment by sircharlo** [comment]
This should be fixed now!
